### DISCUSSION TITLE
Option to send original unmodified email as attachment.  (as .eml file)

### DISCUSSION THREAD

**Discussion by SZemanek** [anonaddy/anonaddy#499] [Feature Requests]
It would be nice if there was the option to forward all emails unmodified as a .eml file, so that all the original unmodified headers and original from email address are there.  

This is beneficial in at least two ways:

1. Often if I'm replying to an email, I usually don't care if I expose my real email address.   I also might not want them to know that I'm using a forwarding service.  For example:  I might get an email from Bob@BobsCompany.com  I might not want them to see:  `From: Bob Syldell 'Bob@BobsCompany.com' <8h76g76g+Bob=BobsCompany.com@anonaddy.me>`  If it gets to the point where I'm replying to an email, I probably don't mind my real email address getting exposed.  

2. My friend who uses anonaddy got a phishing email pretending to be his bank.   For his bank account he used: "SmallCreditUnionName4543@hisdomain.com" as his email address.  Only him and his credit union knew this email address existed, so when he started getting phishing emails, he knew the bank was likely compromised.   So he contacted his banks security dept, and sent them the emails with all the headers.  Although I know all the original header info is there, it still gets heavily modified, and would make it much easier to have the original email before anonaddy modifies it.

**Comment by beedell-roke** [comment]
@SZemanek, I believe that this might be better filed as an issue. I'd like to link this to a few others (although it's already applicable to [some questions](https://github.com/anonaddy/anonaddy/discussions/908#discussion-10416412) of mine).
